src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -44,9 +44,14 @@
     </EnumProperty.DataSource>
   </EnumProperty>
   <StringProperty Name="StartupObject" DisplayName="Type that contains the entry point" />
+  <StringProperty Name="ApplicationManifest" DisplayName="Application Manifest" >
+    <StringProperty.DataSource>
+      <DataSource Persistence="ProjectFileWithInterception" PersistedName="ApplicationManifest" HasConfigurationCondition="false" />
+    </StringProperty.DataSource>
+  </StringProperty>
   <StringProperty Name="Win32ResourceFile" DisplayName="Win32 Resource File">
     <StringProperty.DataSource>
-      <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
+      <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" HasConfigurationCondition="False" />
```

Comment:
It makes it so that when SetValue is called, the value gets persisted to the common PropertyGroup and not to the configuration specific ones (i.e Debug\Release). The old project system cannot handle having different values for some properties based on debug vs release. So for roundtripping reasons putting this in the common propertygroup.